packages/design-system-tailwind-preset/CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0]
+
+### Changed
+
+- Updated to use new font family configuration from @metamask/design-tokens@6.0.0 ([#499](https://github.com/MetaMask/metamask-design-system/pull/499)). Note: This includes breaking changes in the design-tokens package - see the [@metamask/design-tokens migration guide](../design-tokens/MIGRATION.md#from-version-510-to-600) for details.
```

Comment:
This isn’t a breaking change for the Tailwind preset package, as it works with both `@metamask/design-tokens@6.0.0` and all previous versions.  
See: https://github.com/MetaMask/metamask-design-system/pull/499/files#r2008389396